psydac/core/bsplines.py

```diff
@@ -423,6 +433,7 @@ def breakpoints(knots, degree, tol=1e-15, out=None):
     breaks : numpy.ndarray (1D)
         Abscissas of all breakpoints.
     """
+    knots = np.asarray(knots)
```

Comment:
As above

Reply:
Done in https://github.com/pyccel/psydac/commit/54fa1fb4c2afba79c77392308a193bad59161fee